docs/docusaurus.config.js

```diff
@@ -243,11 +279,7 @@ const config = {
             ],
           },
         ],
-        copyright: `
-          <small>
-          Built with Docusaurus on ${docsMetadataJson.site.lastUpdatedAt}
-          </small>
-          `,
+        copyright: `© 2024`,
```

Comment:
Are we legally required to put © somewhere on the page? The project's copyright and license notices would be available on the repository too (but are outdated).

Reply:
As far as I'm aware it's a legal requirement and this was part of the approved design/copy for this piece of work